.github/workflows/node.js.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
```

Comment:
It's minor but we can change this name to `CI` or `Continuous Integration` for better build identification.